applications/nrf5340_audio/src/bluetooth/bt_stream/CMakeLists.txt

```diff
@@ -0,0 +1,16 @@
+#
+# Copyright (c) 2023 Nordic Semiconductor
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+zephyr_library_include_directories(
+    broadcast
+    unicast
+)
+
+
+if (CONFIG_TRANSPORT_BIS)
+	target_sources(app PRIVATE
```

Comment:
Should check CONFIG_AUDIO_DEV here

Reply:
No, it shouldn't matter what device you are. A headset can in theory be a broadcaster as well